README.md

```diff
@@ -13,10 +14,35 @@ cd wasm-glow-ballers/wasm-glow-ballers-js/
 # Package a NodeJS library
 wasm-pack build --target nodejs # [1]
 # Use the NodeJS library to log the mapping where the target is the library itself
-node -e 'console.log(require("./pkg").analyse(require("fs").readFileSync("./pkg/wasm_glow_ballers_js_bg.wasm")))' # [2]
+node -e 'console.log(require("./pkg").analyse(require("fs").readFileSync("../demo/demo.wasm")))' # [2]
```

Comment:
There is both a 'wasm' and 'wat' file present in the demo directory. Preferably there are no binaries present in the repository, making it easier for others to inspect the source code. If they are equivalent, the binary is redundant.